abis/index.ts

```diff
@@ -1,4 +1,5 @@
 import IERC20Metadata from "./IERC20Metadata.json"
+import WAsset from "./WAsset.json"
```

Comment:
Only reason was that the Wasset had the allocateTo function I was using for testing. But we won't need it for the public build

Reply:
@tenax002 Actually I just realized that if we want to include minting functionality on the frontend for Fuji V4, we will need `allocateTo` but I think we can revisit that later maybe since the trading competition won't support short calls anyway